build/common.targets

```diff
@@ -144,10 +144,23 @@
       Properties="Configuration=$(Configuration);
                   VisualStudioVersion=$(VisualStudioVersion);
                   PackageOutputPath=$(NupkgOutputDirectory);
+                  IncludeSymbols=true;
                   NoBuild=true;">
     </MSBuild>
   </Target>
 
+  <!--
+    ============================================================
+    GetPackProject - gets the list of projects need to be packed
```

Comment:
This runs on one project, so this is really a self reporting target. 

The name might be if it started with `Is` instead of Get. 

See example:

https://github.com/NuGet/NuGet.Client/blob/f308eaa17d132d8f9119ff5a672fce117eeed3d9/src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets#L1286-L1299

Reply:
Thanks for reviewing! Fixed.